SPECS/nmi/nmi.spec

```diff
@@ -34,14 +34,12 @@ NMI is the resource that is used when your pods look to use their identity.
 %autosetup -c -N -n %{name}-%{version}
 pushd aad-pod-identity-%{version}
 %patch0 -p1
```

Comment:
unrelated to the changes in this PR but is this still needed? `%autosetup` is already used above

Reply:
It uses `%autosetup -N`, which does makes it not apply patches.